examples/streaming/README.md

```diff
@@ -0,0 +1,164 @@
+
+## Snap Plugin Go Library: Stream Collector Plugin Example
+Here you will find an example plugin that covers the basics for writing a stream collector plugin.
+
+## Plugin Naming, Files, and Directory
```

Comment:
When I saw this section I though on the [PLUGIN_AUTHORING.md](https://github.com/intelsdi-x/snap/blob/master/docs/PLUGIN_AUTHORING.md) page.

But well, it's not important for the purpose of this PR :)